docs/contributing/style_guide.rst

```diff
@@ -24,7 +24,22 @@ code
 * **Experimental methods should log an experimental warning when called**: "Warning: some_method is experimental. Methods, APIs, and core behavior may change in the future."
 * **Experimental classes should log an experimental warning when initialized**: "Warning: great_expectations.some_module.SomeClass is experimental. Methods, APIs, and core behavior may change in the future."
 * **Docstrings are highly recommended**. We use the Sphinx's `Napoleon extension <http://www.sphinx-doc.org/en/master/ext/napoleon.html>`__ to build documentation from Google-style docstrings.
-* **Lint your code**. Our CI system will check using ``black`` and ``isort``. We have a git pre-commit configuration in the repo, so you can just run ``pre-commit install`` to automatically run your changes through the linting process before submitting.
+* **Lint your code**. Our CI system will check using ``black`` and ``isort``.
+  - Linting with ``isort`` *MUST* occur from a virtual environment that has all required packages installed, and pre-commit uses the virtual environment from which it was installed, whether or not that environment is active when making the commit. So, **before running ``pre-commit install`` ensure you have activated a virtual environment that has all development requirements installed**.
+
+    .. code-block:: bash
+
+        pre-commit uninstall
+        # ACTIVATE ENV, e.g.: conda activate pre_commit_env OR source pre_commit_env/bin/activate
+        pip install -r requirements-dev.txt
+        pre-commit install --install-hooks
+
+  - To manually run the linter in your environment, for example if you have already committed files that are rejected by the linting test, you can run tests manually:
```

Comment:
Non-blocking: I read this sentence three times and I still don't understand it.

Reply:
```suggestion
  - If you have already committed files but are seeing errors during the continuous integration tests, you can run tests manually:
```

Reply:
Please commit above if it addresses the question...